Tools/autotest/common.py

```diff
@@ -1624,18 +1624,20 @@ def check_test_syntax(self, test_file):
                 # check for lambda: test_function without paranthesis
                 faulty_strings = re.findall(r"lambda\s*:\s*\w+.\w+\s*\)", f.read())
                 if faulty_strings:
-                    desc = ("Syntax error in autotest lambda at : " +
-                            faulty_strings)
+                    desc = "Syntax error in autotest lambda at : \n"
+                    for x in range(len(faulty_strings)):
+                        desc += faulty_strings[x] + "\n"
                     raise ErrorException(desc)
-        except ErrorException:
-            self.progress('FAILED: "%s"' % "Check for syntax mistake in autotest lambda")
+        except ErrorException as msg:
+            self.progress("FAILED: Check for syntax mistake in autotest lambda. \n" + str(msg))
             exit(1)
-        self.progress('PASSED: "%s"' % "Check for syntax mistake in autotest lambda")
+        self.progress("PASSED: Check for syntax mistake in autotest lambda")
 
     @abc.abstractmethod
```

Comment:
This doesn't look particularly abstract.

Reply:
fixed